ops/model.py

```diff
@@ -1074,7 +1074,6 @@ def is_ready(self) -> '_ContainerReady':
          raising exceptions.
 
         Example:
```

Comment:
This still doesn't render as a monospaced code block in ReST. Use the `::` syntax, so `Example::` followed by a blank line:

```suggestion
        Example::
        
```

Reply:
There's another instance of this (and triple back ticks) below.

In general, the docs need a bit of a syntax go-over, but oh well, another day...

Reply:
BTW, you can build the docs with `./build_docs` (assuming you're in a venv with `requirements-dev.txt` libraries installed).